liberty-maven-plugin/src/it/binary-scanner-it/pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.openliberty.tools.it</groupId>
+    <artifactId>binary-scanner-it</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13.1</version>
+        </dependency>
+        <dependency>
+            <groupId>io.openliberty.tools</groupId>
+            <artifactId>liberty-maven-plugin</artifactId>
+            <version>@pom.version@</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-httpclient</groupId>
+            <artifactId>commons-httpclient</artifactId>
+            <version>3.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.10.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.plexus</groupId>
+            <artifactId>plexus-utils</artifactId>
+            <version>3.4.2</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>2.19.1</version>
+                <configuration>
+                    <redirectTestOutputToFile>true</redirectTestOutputToFile>
+                    <forkMode>once</forkMode>
+                    <forkedProcessTimeoutInSeconds>1200</forkedProcessTimeoutInSeconds>
+                    <argLine>-enableassertions</argLine>
+                    <workingDirectory>${project.build.directory}</workingDirectory>
+                    <includes>
+                        <include>**/*Test.java</include>
+                    </includes>
+                    <systemPropertyVariables>
+                        <mavenPluginVersion>@pom.version@</mavenPluginVersion>
+                        <runtimeVersion>${runtimeVersion}</runtimeVersion>
+                        <!-- Scanner version e.g. 22.0.0.3 or 22.0.0.3-SNAPSHOT -->
+                        <scannerVersion>22.0.0.3-SNAPSHOT</scannerVersion>
```

Comment:
Sorry should have brought this up in the last PR. Should we set this scanner version to a variable similar to how `${runtimeVersion}` is configured and then set via the GH actions workflows? https://github.com/OpenLiberty/ci.maven/blob/79ef5c2e9a6233390c437c15bbcc242a76e9d5ec/.github/workflows/maven.yml#L25

Reply:
Though for us since we are starting with running this suite locally, we would set it via a Maven parameter ie. `mvn verify .... -DscannerVersion=22.0.0.3-SNAPSHOT`

Reply:
This is something that we discussed at the beginning and decided just to hard code this value here for several reasons. Passing the value in would require quite a few changes to pass the value from one level to the next. We only expect to run the test suite when there is a new binary scanner which is not very often so that is a lot of complexity for something we rarely use. Also we expect only to run it manually so adding the expected version number to a file is easy. The main problem is documenting how to run the suite properly. The comment there intends to show the two expected formats.

Reply:
Makes sense, thanks